src/open_inwoner/cms/open_product/product_list/tests/test_product_list_plugin.py

```diff
@@ -0,0 +1,83 @@
+from unittest.mock import MagicMock, patch
+
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+
+from open_inwoner.cms.open_product.product_list.models import (
+    ProductList as ProductListModel,
+)
+
+
+@patch("open_inwoner.cms.open_product.product_list.cms_plugins.get_open_product_client")
+class ProductListTestCase(CMSTestCase):
+    def setUp(self):
+        self.page = create_page("Product List test Page", "cms/fullwidth.html", "nl")
+        self.placeholder = self.page.placeholders.get(slot="content")
+
+        self.product_list = ProductListModel.objects.create(
+            title="Mijn reisdocumenten",
+            description="Some description",
```

Comment:
```suggestion
            description="Een beschrijving",
```
or some other Dutch phrase to make it uniform